### PR TITLE
Dmirano/3782 proposed budget restructure

### LINKED ISSUE #3782: [Feature] move all proposed budget components into a proposed-budget folder
**Description and related issues -**
Create a proposed-budget folder underneath the apd folder and move the following components into it.

```
├── proposed-budget
            ├── ProposedBudget
            ├── BudgetSummary
            ├── IncentivePayments
            ├── QuarterlyBudgetSummary
            ├── SummaryBudgetByActivity
            ├── SummaryActivityBreakdown
            ├── (viewOnly)ProposedBudget rename to ProposedBudgetReadOnly```

### This task is done when…
- [ ] all of the components listed above are in the proposed-budget folder
- [ ] rename view only ScheduleSummary to ScheduleSummaryReadOnly

### PR DESCRIPTION
**Pull request by mirano-darren**
Resolves #3782 

**Description-**
proposed budget refactor

**This pull request changes...**

- file locations of requested filed and affected tests

**This pull request was tested in the follow ways…**

- npm test in /web/ passes
- app boots up normally

**Steps to manually verify this change...**

1. verify all requested files were moved to the right location

### This pull request is ready to review when...

- [x] Automated tests are updated (and all tests are passing)
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [x] Code has been reviewed by someone other than the original author
- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by mirano-darren** [comment]
merge conflicts will be solved once executive summary refactor is in (#3912)

File: web/src/pages/apd/ApdReadOnly.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ import ApdSummary from '../../containers/viewOnly/ApdSummary';
 import PreviousActivities from './previous-activities/PreviousActivities';
 import Activities from '../../containers/viewOnly/activities/All';
 import ScheduleSummary from './schedule-summary/ScheduleSummaryReadOnly';
-import ProposedBudget from '../../containers/viewOnly/ProposedBudget';
+import ProposedBudget from './proposed-budget/ProposedBudget';
```
Comment (thetif) [inline web/src/pages/apd/ApdReadOnly.js:20]: I think you are using the wrong one here, should be ProposedBudgetReadOnly

**Review by thetif — CHANGES_REQUESTED** [review]
one file needs to be switched

File: web/src/pages/apd/ApdReadOnly.js (hunk 1 above)
Comment (mirano-darren) [reply web/src/pages/apd/ApdReadOnly.js]: ahhh!!! keep making the same mistake. will make sure to double check this once I create the PR :)

Comment (mirano-darren) [reply web/src/pages/apd/ApdReadOnly.js]: just pushed up the fix

**Review by thetif — APPROVED** [review]

**Pull request merged**